### PR TITLE
Include miliseconds when creating datetimes

### PR DESCRIPTION
**Pull request by mayerm94**
Milliseconds was getting dropped in DateTime construction.

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1974
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated NA
- [x] Update cqm-execution test fixtures using update_cqm_execution_fixtures.sh script from server-scripts repo

**Bonnie Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/cqm/converter/utils.rb [hunk 1]
```diff
@@ -141,7 +141,7 @@ def self.hds_to_qdm_js(js_dependencies, record, qdm_model_attrs)
     # Adjust improper date times from the cql_qdm_patientapi.
     def self.date_time_adjuster(results)
       if results.is_a?(Hash) && results.key?('year') && results.key?('minute')
-        DateTime.new(results['year'], results['month'], results['day'], results['hour'], results['minute'], results['second'], results['millisecond']).to_s
+        DateTime.new(results['year'], results['month'], results['day'], results['hour'], results['minute'], (results['second'] + (results['millisecond'] / 1000.0))).strftime('%Y-%m-%d %H:%M:%S.%N')
```
Comment (losborne) [inline lib/cqm/converter/utils.rb]: I believe the format string needs to be `%Y-%m-%d %H:%M:%S.%N%Z` in order to include the timezone offset
